kitsune/users/migrations/0005_set_initial_contrib_email_flag.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+Set the `first_{revision,question}_email_sent` flags for existing users, to
+avoid back filling welcome emails to contributors.
+"""
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations
+from django.db.models import F
+
+
+def contrib_email_flags_forwards(apps, schema_editor):
+    Profile = apps.get_model('users', 'Profile')
+    Answer = apps.get_model('questions', 'Answer')
+    Revision = apps.get_model('wiki', 'Revision')
+
+    l10n_contributor_ids = set(
+        Revision.objects
+        .exclude(document__locale='en-US')
```

Comment:
In `settings.py` there is this comment: `# Do not change this without also deleting all wiki documents:`. Given that, I think it is safe to rely on this being constant within a given instance of Kitsune.